### PR TITLE
feat: interfaces added

### PR DESCRIPTION
**Pull request by tanyushanel**

### CODE REVIEW

File: exadel-trainee-app/src/app/interfaces/role.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export enum Role {
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/interfaces/role.ts:1]: Could you please check app/route/constants file 
We need to have one Role interface

File: exadel-trainee-app/package.json [hunk 2]
```diff
@@ -70,7 +69,8 @@
   },
   "lint-staged": {
     "*.{js,ts}": [
-      "npm run lint:fix && lint:styles",
```
Comment (RomPavel) [inline exadel-trainee-app/package.json:73]: Could you please explain this change

Comment (tanyushanel) [reply exadel-trainee-app/package.json:73]: Nope. Had problems with Husky today morning. Some adjustments. Shouldn't commit it.

File: exadel-trainee-app/src/app/interfaces/test.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    assignmentEndDate: string;
+  };
+  content: {
+    grammar: TestModule;
+    audition: TestModule;
+    essay: TestModule;
+    speaking: TestModule;
+  };
+  feedback?: string;
+}
```
Comment (Inunor) [inline exadel-trainee-app/src/app/interfaces/test.ts:19]: Looks much better)

File: exadel-trainee-app/src/app/interfaces/test-module.ts [hunk 4]
```diff
@@ -0,0 +1,4 @@
+export interface TestModule {
```
Comment (Inunor) [inline exadel-trainee-app/src/app/interfaces/test-module.ts:1]: You're using just id in other interfaces, so for consistency, I would suggest using testId -> id, testMark - mark

**Review by Inunor — APPROVED** [review]

**Pull request merged**